### DISCUSSION TITLE
Add option to choose a cookies file

### DISCUSSION THREAD

**Discussion by ZzTriplett** [aandrew-me/ytDownloader#202] [Ideas]
Would there be a way to add the option of selecting a cookies.txt file to use rather than using the cookies from browser option? I have issues with needing cookies for certain downloads and the cookies from browser option doesn't work well with flatpak (yes, I've used flatseal to try to give access to the file system and still no luck). 

You would use "--cookies /path/to/cookies/file.txt" in place of "--cookies-from-browser browser" in cookieArg with an option to select the file in the preferences page.

**Comment by aandrew-me** [comment]
Since you're using the flatpak app, your browser also needs to be installed with flatpak for that option to work.
I will try to add option to use cookies from file
